.travis.yml

```diff
@@ -8,25 +8,31 @@ cache:
         - $HOME/.composer/cache/files
 
 env:
-    globals:
-        - deps=no
+    - DEPS=no
 
 before_install:
     - phpenv config-rm xdebug.ini
 
 before_script:
-    - if [ "$deps" = "low" ]; then composer --prefer-lowest --prefer-stable update; fi;
-    - if [ "$deps" = "no" ]; then composer install; fi;
+    - if [ "$DEPS" == "low" ]; then composer --prefer-lowest --prefer-stable update; fi;
+    - if [ "$DEPS" == "no" ]; then composer install; fi;
 
-script: ./vendor/bin/simple-phpunit
+script: |
+    if [[ $TRAVIS_PHP_VERSION = 7.* || $TRAVIS_PHP_VERSION = nightly ]]; then
+        SYMFONY_PHPUNIT_VERSION=6.1 ./vendor/bin/simple-phpunit
```

Comment:
I would not force using 6.1 as this is EOLed. The bridge now already use PHPUnit 6 on PHP 7.2 where it is needed